rmw_opendds_cpp/src/rmw_publisher.cpp

```diff
@@ -307,7 +283,7 @@ rmw_publisher_count_matched_subscriptions(
   size_t * subscription_count)
 {
   if (!publisher) {
-    RMW_SET_ERROR_MSG("publisher handle is null");
+    RMW_SET_ERROR_MSG("publisher is null");
```

Comment:
Should we use macro `RMW_CHECK_ARGUMENT_FOR_NULL` to check for null arguments in all places for consistency?

Reply:
Yes.